alumnos/piero/service.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+start() {
+  awk 'NR==1 {print $1}' /mnt/request.log
+}
+
+help() {
+   echo "Usage: $0 {start|help}"
+}
+
+
+case "$1" in
+start)
+start
+;;
+*)
```

Comment:
As per instructions, there should be present also the option "HELP"

Reply:
ty :)